search/index.html

```diff
@@ -53,4 +53,7 @@
     {{/flash}}
   </div>
 </script>
+<script id="emoji-template" type="x-tmpl-mustache">
+<img class="emoji" title=":{{code}}:" alt=":{{code}}:" src="https://assets-cdn.github.com/images/icons/emoji/{{code}}.png" height="20" width="20" align="absmiddle"/>
```

Comment:
I've misread this on first commit, thought it was a class.

> The align attribute of <img> is not supported in HTML5

See e.g. http://www.w3schools.com/tags/att_img_align.asp